test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -156,6 +194,31 @@ pollWithRetry env rks = do
                   putStrLn $ "poll succeeded after " ++ show (maxSendRetries - retries) ++ " retries"
                   return result
 
+maxMemPoolRetries :: Int
+maxMemPoolRetries = 30
+
+mpLookupWithRetry
+    :: MempoolLookupApiCmd
+    -> ClientEnv
+    -> [TransactionHash]
+    -> IO (Either ServantError [LookupResult ChainwebTransaction])
+mpLookupWithRetry cmd env txs = do
```

Comment:
Test will go faster if you invoke `/lookup` in a loop instead of `/poll`. Are we testing `/listen` yet?

Reply:
no, mostly since I wasn't sure of the listen timeout and wanted it to be short as possible for the tests. Should I try /listen instead?  Or both I suppose...